rbd/rbd.go

```diff
@@ -916,6 +916,22 @@ func (image *Image) GetId() (string, error) {
 
 }
 
+// Set updates the rbd image (not the Snapshot) such that the snapshot
```

Comment:
The doc comment should start with the name of the function `SetSnapshot`. We can drop the "(not the Snapshot)" bit because that was added to `snasphot.Set` to clarify it's (bad) behavior.